src/schema/bidder_position.js

```diff
@@ -119,6 +119,10 @@ const BidderPositionType = new GraphQLObjectType({
       type: GraphQLInt,
       deprecationReason: "Favor `suggested_next_bid`",
     },
+    message: {
+      type: GraphQLString,
```

Comment:
Just pinged @briansw about clarifying how we want to display the error messages, but I think we should handle mapping the gravity errors --> user-friendly copy _here_ so we don't have to replicate in clients.